### PR TITLE
Generated Visual Recognition with Adapter

### PR DESCRIPTION
**Pull request by AmmarD212**
##  Notes/Considerations

- Tests were updated to use the new name i.e `watson_vision_combined`.
- BaseService was updated to pick up `api_keys`
- Style updates for core-sdk.

### CODE REVIEW

File: lib/base_service.ts [hunk 1]
```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  */
 
- 'use strict';
+'use strict';
```
Comment (germanattanasio) [inline lib/base_service.ts]: do we need the `use strict` ?

File: test/unit/test.visual_recognition.v3.js [hunk 2]
```diff
@@ -84,7 +84,7 @@ describe('visual_recognition', function() {
     nock.cleanAll();
   });
 
-  const visual_recognition = watson.visual_recognition(service);
+  const visual_recognition = watson.watson_vision_combined(service);
```
Comment (germanattanasio) [inline test/unit/test.visual_recognition.v3.js]: why?

Comment (AmmarD212) [reply test/unit/test.visual_recognition.v3.js]: According to swagger: `"x-vcap-service-name": "watson_vision_combined"`

File: lib/base_service.ts (hunk 1 above)
Comment (AmmarD212) [reply lib/base_service.ts]: This doesn't need to be here. I can configure the `tsc` compiler to emit `use strict` in the .js files.

**Pull request merged**